website/content/docs/lambda/registration/automate.mdx

```diff
@@ -0,0 +1,190 @@
+---
+layout: docs
+page_title: Automate Lambda Function Registeration
+description: >-
+  Register AWS Lambda functions with Consul service mesh using the Consul Lambda registrator. The Consul Lambda registrator automates Lambda function registration.
+---
+
+# Automate Lambda Function Registeration
+
+This topic describes how to automate Lambda function registration using the Consul Lambda registrator module for Terraform.
```

Comment:
This page toggles between putting "the" before "Consul Lambda registrator" and "Lambda registrator" and not having it. I've never referred to it as "the Lambda registrator" before, but I think the most important thing is consistency.